testing/tests/DevExpress.ui/themes.tests.js

```diff
@@ -568,17 +568,6 @@ QUnit.module("web font checker", () => {
         assert.equal(diff, 0, "Element's count are the same after method call");
     });
 
-    test("waitWebFont: function reject by timeout if the font is not loaded", (assert) => {
```

Comment:
I suggest rewriting this test to make sure that promise will be resolved if the font is not loaded.